src/web/client/test/unit/extension.test.ts

```diff
@@ -4,15 +4,9 @@
  */
 
 import { expect } from "chai";
-import { convertfromBase64ToString, convertStringtoBase64 } from "../../utilities/commonUtil";
 
-describe("Web Extension Unit Tests", async () => {
-    it("it should return text from base 64", () => {
-        const str = "this is test case";
-
-        const base64 = convertStringtoBase64(str);
-        const base64totext = convertfromBase64ToString(base64);
-        expect(base64).eq("dGhpcyBpcyB0ZXN0IGNhc2U=");
-        expect(base64totext).eq(str);
+describe("test", () => {
+    it("testing", () => {
+        expect(1).eq(1);
```

Comment:
Is this some placeholder?

Reply:
yes. Initially i removed this file but this file is being used in config so i added and move test cases to it's appropriate location